mainapp/proj.android-studio/app/src/org/cocos2dx/cpp/AppActivity.java

```diff
@@ -94,6 +100,17 @@ public static void clearLaunchString() {
         _launchString = "";
     }
 
+    public Cocos2dxGLSurfaceView onCreateView() {
+        glSurfaceView = new Cocos2dxGLSurfaceView(this);
+
+        this.hideSystemUI();
+
+        // create stencil buffer
+        glSurfaceView.setEGLConfigChooser(5, 6, 5, 0, 16, 8);
```

Comment:
Add comment for the hardcoded integer parameters

Reply:
I did not put this line. Parts of the previous code were automatically removed and then added again when I was trying to resolve the conflicts.